src/components/sbb-card/sbb-card.tsx

```diff
@@ -0,0 +1,51 @@
+import { Component, h, Host, JSX, Prop, State } from '@stencil/core';
+import { InterfaceSbbCardAttributes } from './sbb-card.custom';
+
+/**
+ * @slot unnamed - Slot to render the content.
+ * @slot badge - Slot to render `<sbb-card-badge>`.
+ */
+
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-card.scss',
+  tag: 'sbb-card',
+})
+export class SbbCard {
+  /**
+   * Size variant, either xs, s, m, l, xl and xxl.
+   */
+  @Prop() public size?: InterfaceSbbCardAttributes['size'] = 'm';
+
+  /* @internal */
+  @State() public hasBadge = false;
+
+  /**
+   * It is used internally to show the `<sbb-card>`.
+   *
+   * @param size The size selected.
+   * @returns True whether size is equal to m, l, xl or xxl. False otherwise.
+   */
+  private static _showSBBBadge(size: string): boolean {
+    return size === 'm' || size === 'l' || size === 'xl' || size === 'xxl';
+  }
+
+  private _onSlotBadgeChange(): void {
```

Comment:
This could probably be inlined. It also does not consider the situation, that a badge might be removed.
See https://github.com/lyne-design-system/lyne-components/blob/master/src/components/sbb-form-field/sbb-form-field.tsx#L178-L181 for comparison.